src/build123d/geometry.py

```diff
@@ -445,10 +445,14 @@ def project_to_line(self, line: Vector) -> Vector:
 
         return line * (self.dot(line) / (line_length * line_length))
 
-    def distance_to_plane(self):
-        """Minimum distance between vector and plane"""
-        raise NotImplementedError("Have not needed this yet, but OCCT supports it!")
-
+    def distance_to_plane(self, plane: Plane) -> float:
+        """Minimum unsigned distance between vector and plane"""
+        return plane.wrapped.Distance(self.to_pnt())
+
+    def signed_distance_from_plane(point: Vector, plane: Plane) -> float:
```

Comment:
```suggestion
    def signed_distance_from_plane(self, plane: Plane) -> float:
```

Reply:
Wow, Python let me get away with that.  I've fixed the param name in the declaration and the body.